common/traits/wc_race_traits.txt

```diff
@@ -1978,6 +1978,23 @@ creature_ancient_wind = {
 	random = no
 	customizer = no
 }
+creature_ancient_coral = {
```

Comment:
The trait isn't added in triggers where other ancients are mentioned like `has_race_trait_trigger`.